src/components/programs-list/tests/__snapshots__/ProgramListPage.test.jsx.snap

```diff
@@ -836,7 +822,7 @@ exports[`ProgramListPage renders fetching program error page when there are issu
         <p>
           © 
           2022
-           edX Inc. All rights reserved.
```

Comment:
2 tests were failing after migrating the `eslint` package and I had to update the snapshots to get those tests passing. These changes to the snapshot look trivial to me.